packages/router/src/page-loader.js

```diff
@@ -2,6 +2,8 @@ import { useContext } from 'react'
 
 import isEqual from 'lodash.isequal'
 
+import { isPrerendering } from '@redwoodjs/web'
```

Comment:
Check how I did the export for `@redwoodjs/web/apollo`, I would love to have something like `@redwoodjs/prerender/browser` instead of re-exporting these in "web."

Reply:
Mhmm... tried doing it the same way, but remember that we need these util functions both internally and externally.

Externally i.e. by the user, would be fine using the webpack changes like in the apollo export.

Internally i.e. within the RW codebase, webpack wouldn't be loaded, so might be an issue.

I'm totally agreed with you on this point, but webpack/typescipt/config seems to be getting in the way. How do you feel about keeping this issue separate?